packages/analytics/analytics-chart/sandbox/pages/TimeSeriesChartDemo.vue

```diff
@@ -296,7 +296,7 @@ const metricItems = [{
 
 // Short labels
 const statusCodeLabels = [
-  '200', '300', '400', '500',
+  '200', '300', '400', '500', '5XX',
```

Comment:
Why only `5XX` here, and not other grouped status codes as well?

Reply:
This is just the sandbox...  just testing stuff out

Reply:
oh my bad. Disregard :)